### PR TITLE
Plone 6

### PR DESCRIPTION
**Pull request by petschki**

### CODE REVIEW

File: Gruntfile.js
Comment (rnixx) [inline Gruntfile.js:1]: Do you plan to port this package to scss as you did with b.p.cart?

File: package-lock.json
Comment (rnixx) [inline package-lock.json:1]: Do we want to include package-lock.json?

File: src/bda/plone/checkout/browser/form.py [hunk 1]
```diff
@@ -46,14 +46,32 @@ def __iter__(self):
...
+SVG_NEXT = """<svg class="bi bi-chevron-right" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http: // www.w3.org/2000/svg">
+                <path fill-rule = "evenodd" d = "M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"/>
+              </svg >
+           """
+SVG_FINISH = """<svg class="bi bi-credit-card" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" d="M14 3H2a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1zM2 2a2 2 0 0 0-2 2v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V4a2 2 0 0 0-2-2H2z"/>
+                  <rect width="3" height="3" x="2" y="9" rx="1"/>
+                  <path d="M1 5h14v2H1z"/>
+                </svg>
+             """
```
Comment (rnixx) [inline src/bda/plone/checkout/browser/form.py:63]: The formatting of the inline svg is somewhat adventurous.

**Review by rnixx — COMMENTED** [review]
Overall LGTM

File: Gruntfile.js
Comment (petschki) [reply Gruntfile.js:1]: done

File: package-lock.json
Comment (petschki) [reply package-lock.json:1]: I always commit the lock files (yarn, npm)  in order to pin the installed versions (like constraints.txt in python) ... this is also best practice in mockup development.

File: src/bda/plone/checkout/browser/form.py (hunk 1 above)
Comment (petschki) [reply src/bda/plone/checkout/browser/form.py:63]: @agitator  😁

File: package-lock.json
Comment (rnixx) [reply package-lock.json:1]: ok

**Pull request merged**